views/home_page.hbs

```diff
@@ -55,3 +57,57 @@
     </div>
   </div>
 </footer>
+
+
+
+<!-- Log In Modal -->
+<div id="login" class="modal">
+  <div class="modal-content">
+    <h2 class="right-align">
+      <a class="modal-close btn-large btn-floating btn-flat indigo lighten-2 white-text" href="#!" id="close">&times;</a>
+    </h2>
+    <h3 class="header center indigo-text text-lighten-2">Account Login</h3>
+  <form class="modal-content animate" action="/action_page.php">
+    <div class="container">
+      <br>
+      <label for="email"><b>Email</b></label>
+      <input class="grey lighten-1" type="text" name="email" id="email" required>
+      <button class="btn btn-login indigo lighten-2" id="loginbutton" name="login" type="submit">Welcome Back</button>
+      <a class="modal-close modal-trigger" href="#signup" id="close">Don't have an account? Sign Up</a>
+    </div>
+  </form>
+  </div>
+</div>
+
+<!-- Sign Up Modal -->
+<div id="signup" class="modal">
```

Comment:
I like the idea of having a signup modal, but what about the `/signin` and `/register` pages the @thegeophysicist worked on prior? How would this integrate with those?

Reply:
I created the `/signin` and `/register` pages as a way to show the signin and register functionality.  I think we can use event listeners in the modal to call the functions.

Reply:
Taking care of integration now!